includes/Core/Modules/Modules.php

```diff
@@ -882,6 +900,11 @@ private function get_module_schema() {
 					),
 					'readonly'    => true,
 				),
+				'owner'        => array(
```

Comment:
Maybe we could bake the `list_users` permission check into here instead of JS (see my other comment above)? That would actually simplify things a bit, plus it would ensure we don't even surface this data to users that shouldn't see it.

Since that is based on user permissions, I'd argue here we could alter even the base response: The `owner` property would only be included (either as object or `null`) if the current user can `list_users`.

Reply:
Yes, a good idea. Added capability check to the Modules class. Thanks!